Resources/Prototypes/Loadouts/items.yml

```diff
@@ -183,6 +183,22 @@
   items:
     - CrowbarRed
 
+# Plushies
+- type: loadout
+  id: LoadoutItemPlushieLizard
+  category: Items
+  cost: 2
+  items:
+    - PlushieLizard
+
+- type: loadout
+  id: LoadoutItemPlushieSpaceLizard
+  category: Items
+  cost: 2
+  items:
+    - PlushieSpaceLizard
```

Comment:
```suggestion
```
Plushies aren't actually a good fit for loadouts, largely because they're sort of wasted filler items that normally get mapped anyway. We generally want to limit the Items tab to items that you can actually use, or serve some kind of purpose. Crayons oddly enough are fine!